_posts/2025-03-12-superfast-debezium.adoc

```diff
@@ -0,0 +1,672 @@
+---
+layout: post
+title:  Build a Superfast CDC with Debezium, Quarkus and GraalVM
+date:   2025-03-12 10:11:11 +0100
+tags: [ debezium, quarkus, graalvm, performance ]
+author: gpanice
+---
+
+Change Data Capture (CDC) is widely used in various contexts, such as microservices communication, legacy system modernization, and cache invalidation. The core idea of this pattern is to detect and track changes in a data source (e.g., a database) and propagate them to other systems in https://www.reddit.com/r/embedded/comments/1ddbul4/people_always_abuse_the_term_realtime_but_what/[real-time] or near real-time. Debezium is a CDC platform that provides a wide range of connectors for most data sources. Beyond capturing changes, it also offers transformation capabilities through an intuitive UI for defining debezium instances.
```

Comment:
Hi @kmos, one thing you'll find we try to do as much as possible in our .adoc files is to maintain one sentence per line. This makes it much easier for diffs and things like that.  No need to worry with it here, but just something to keep in mind for next time.